changelog.d/20231023_132746_maria_limit_one_user_to_one_task_at_a_time.md

```diff
@@ -0,0 +1,15 @@
+### Added
+
+- Ability to limit one user to one task at a time
+  (<https://github.com/opencv/cvat/pull/6975>)
+
+### Fixed
+
+- Bug with viewing dependent rq jobs for downloading resources from
+cloud storage when file path contains sub-directories
```

Comment:
Could you make this description more clear for the platform users? It's quite hard to understand what is happening in this message.